docs/logo-small.svg

```diff
@@ -0,0 +1,146 @@
+   y="0px"
+   width="1200"
+   height="1200"
+   viewBox="0 0 1200 1200"
+   xml:space="preserve"
+   id="svg12347"
+   sodipodi:docname="Signals.svg"
+   inkscape:version="1.2 (dc2aeda, 2022-05-15)"
+   inkscape:export-filename="../Desktop/Signals.svg"
+   inkscape:export-xdpi="96"
+   inkscape:export-ydpi="96"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:svg="http://www.w3.org/2000/svg"><defs
+   id="defs12351" /><sodipodi:namedview
+   id="namedview12349"
+   pagecolor="#ffffff"
+   bordercolor="#000000"
+   borderopacity="0.25"
+   inkscape:showpageshadow="2"
+   inkscape:pageopacity="0.0"
+   inkscape:pagecheckerboard="0"
+   inkscape:deskcolor="#d1d1d1"
+   showgrid="false"
+   inkscape:zoom="0.19124081"
+   inkscape:cx="985.66827"
+   inkscape:cy="1511.1837"
+   inkscape:window-width="1512"
+   inkscape:window-height="836"
+   inkscape:window-x="0"
+   inkscape:window-y="38"
+   inkscape:window-maximized="0"
+   inkscape:current-layer="svg12347" />
+<style
+   type="text/css"
+   id="style12274">
+	.st0{display:none;}
+	.st1{fill:#F15A24;stroke:#000000;stroke-miterlimit:10;}
+	.st2{stroke:#000000;stroke-miterlimit:10;}
+	.st3{fill:none;stroke:#000000;stroke-width:60;stroke-linecap:round;stroke-miterlimit:10;}
+	.st4{fill:none;stroke:#000000;stroke-width:51;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:10;}
+	.st5{fill:none;stroke:#000000;stroke-width:71;stroke-linecap:round;stroke-miterlimit:10;}
+	.st6{fill:none;stroke:#000000;stroke-width:101;stroke-linecap:round;stroke-miterlimit:10;}
+	.st7{fill:none;stroke:#000000;stroke-width:149;stroke-linecap:round;stroke-miterlimit:10;}
+	.st8{fill:none;stroke:#000000;stroke-width:100;stroke-linecap:round;stroke-miterlimit:10;}
+	.st9{fill:none;}
+	.st10{fill:#FFFFFF;stroke:#FFFFFF;stroke-width:16;stroke-miterlimit:10;}
+	.st11{fill:#F15A24;stroke:#F15A24;stroke-width:67;stroke-miterlimit:10;}
+	.st12{fill:#F15A24;}
+	.st13{display:none;fill:#FFFFFF;stroke:#000000;stroke-width:67;stroke-miterlimit:10;}
+	.st14{fill:#FFFFFF;stroke:#000000;stroke-width:67;stroke-miterlimit:10;}
+	.st15{fill:#B32B2B;}
+	.st16{fill:#F0A824;}
+	.st17{fill:#76B543;}
+</style>
+
+
+
+
+
+
+
+
+
+
+
+<g
+   id="g15079"
+   inkscape:label="square"
+   transform="translate(-1094.5685,-505.4672)"
+   inkscape:export-filename="../Desktop/square-signals.svg"
+   inkscape:export-xdpi="96"
+   inkscape:export-ydpi="96"
+   style="display:inline"><rect
+     style="fill:#f15a24;fill-opacity:1;stroke:none;stroke-width:21.3333;stroke-linecap:round;stroke-miterlimit:4.1;stroke-opacity:1"
+     id="rect15050"
+     width="1200"
+     height="1200"
+     x="1094.5685"
+     y="505.46719" /><g
+     id="g14996"
+     inkscape:label="trafficlight"
+     transform="matrix(0.60314593,0,0,0.60314593,801.19099,439.09109)"
+     inkscape:export-filename="../Desktop/signals.svg"
```

Comment:
would be cool too clean up this  useless inkscape, sodipodi, Adobe Illustrator properties